e2e/uimodel/installer/installer.go

```diff
@@ -179,7 +180,6 @@ func (i *Installer) IsCreateSiteStep() bool {
 
 // IsInProgressStep checks if installer is in progress
 func (i *Installer) IsInProgressStep() bool {
-	log.Infof("checking if installation is in progress")
```

Comment:
Why removing it from here? It is useful to show these messages to indicate what's going on which is useful when run in headless browser mode.

Reply:
Because log line `log.Info("checking if installation has failed")` will provide information about next step and there is no difference from user side will it fail inside `IsInProgressStep` or other function. Also if it fails we will see stacktrace, which helps to understand which function was failed.